stylesheets/components/avatar.css

```diff
@@ -0,0 +1,30 @@
+.avatar {
+    border: 1px solid var(--gray-500);
+    border-radius: 50%;
+    margin: 1.5rem;
+}
+
+.square-avatar {
+    border-radius: 0.5rem;
```

Comment:
Just a suggestion to give border-radius in percentage, everything else looks good

Reply:
Thanks for suggesting, changed border-radius to percent.